### PR TITLE
fixes #5

### LINKED ISSUE #5: Hard Coded Version in install.js
The version that `install.js` pulls down is [`hard coded`](https://github.com/mafintosh/atom-shell/blob/master/install.js#L11) to `0.18.0`, but the latest release is `0.19.5` which has lots of bug fixes. Maybe we could figure out what the latest version is and have it pull that?

### PR DESCRIPTION
**Pull request by hij1nx**
a quick fix for ensuring that the release downloaded is the latest.

### CODE REVIEW

**Comment by jprichardson** [comment]
Yeah, love this proposal. @mafintosh thoughts?

**Comment by mafintosh** [comment]
I'm all for this. Sorry for being slow. @hij1nx could you resolve the conflict so I can merge this? :)

File: install.js [hunk 1]
```diff
@@ -2,49 +2,102 @@
...
 var platform = os.platform()
-var arch = os.arch()
-if ('win32' === platform) {
-  // 64-bit is not available under windows.
-  arch = 'ia32';
-}
-var version = '0.19.5'
-var name = 'atom-shell-v'+version+'-'+platform+'-'+arch+'.zip'
-var url = 'https://github.com/atom/atom-shell/releases/download/v'+version+'/atom-shell-v'+version+'-'+platform+'-'+arch+'.zip'
+var arch = (platform == 'win32') ? 'ia32' : os.arch()
```
Comment (sindresorhus) [inline install.js]:
```js
var arch = platform === 'win32' ? 'ia32' : os.arch()
```

File: install.js [hunk 2]
```diff
@@ -2,49 +2,102 @@
...
+
+  var req = {
+    'host': 'api.github.com',
+    'path': '/repos/atom/atom-shell/tags',
+    'headers': {
+      'User-Agent': 'Node'
+    }
+  }
+
+  https.get(req, function(res) {
```
Comment (sindresorhus) [inline install.js]: You could use https://github.com/sindresorhus/gh-got to DRY this up.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "atom-shell",
-  "version": "1.2.1",
+  "version": "2.0.0",
```
Comment (sindresorhus) [inline package.json]: Don't bump the version in other peoples projects. It should be up to them what to release it as and when.

**Comment by sindresorhus** [comment]
Would also be nice if you could give this pull request a descriptive title.

File: install.js (hunk 1 above)
Comment (hij1nx) [inline install.js]: platform is always a string, so `===` is pointless. parens for readability.

File: package.json (hunk 3 above)
Comment (hij1nx) [inline package.json]: needs to be bumped though, it opens the conversation.

**Comment by hij1nx** [comment]
lol, just realized this is a dead PR anyway. ffs.

File: install.js (hunk 2 above)
Comment (hij1nx) [inline install.js]: how about a dump truck to move a grain of sand.

**Pull request closed without merging**